data/json/npcs/talk_tags.json

```diff
@@ -620,7 +728,9 @@
       "I'm not a contortionist!",
       "I can't fit in your tiny human vehicle.",
       "This sucks.  I'm too big.",
-      "I don't like being a mutant crammed into your tiny little vehicle."
+      "I don't like being a mutant crammed into your tiny little vehicle.",
+      "Getting awfully claustrophobic in here.",
```

Comment:
This isn't a snippet for when there are too many people in a vehicle, this is a snippet for when the NPC has mutated anatomy and is too large to comfortably fit inside, causing them pain.